src/Console/PrintSchemaCommand.php

```diff
@@ -16,10 +14,8 @@ class PrintSchemaCommand extends Command
      *
      * @var string
      */
-    protected $signature = '
-        lighthouse:print-schema
-        {--W|write : Write the output to a file}
-    ';
+    protected $signature = ' lighthouse:print-schema {--W|write : Write the output to a file}';
```

Comment:
Let's keep this as is, the multi-line way is more readable as it looks like the output from running the help.

Reply:
See https://github.com/nuwave/lighthouse/pull/527/commits/391b756e19a83325c9ccc2525bda89d849eb7fdd @spawnia